### PR TITLE
Replace vegetable oil with canola oil

### PR DESCRIPTION
**Pull request by denis-galajda**

### CODE REVIEW

File: cake-recipe.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup vegetable oil
+ - 1/2 cup canols oil
```
Comment (ATP93) [inline cake-recipe.txt]: typo in canola

Comment (denis-galajda) [inline cake-recipe.txt]: typo in canola

**Review by denis-galajda — COMMENTED** [review]
fix the typo

**Comment by denis-galajda** [comment]
Pull request updated

**Pull request merged**